ddmrp/views/mrp_production_view.xml

```diff
@@ -8,11 +8,13 @@
         <field name="arch" type="xml">
             <field name="date_planned_start" position="before">
                 <field name="execution_priority_level"
-                       bg_color="red:execution_priority_level=='1_red';yellow:execution_priority_level=='2_yellow';green:execution_priority_level == '3_green'"
-                       fg_color="white:execution_priority_level=='1_red';black:execution_priority_level=='2_yellow';white:execution_priority_level == '3_green'"/>
+                       options="{
```

Comment:
would be better to use single quote to wrap and double quotes inside, to make this JSON compat OOTB

Reply:
It's not JSON but a python dict evaluated with pyeval so not sure its useful?